frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```
@@ -78,7 +78,10 @@ const condenseDeviceUsers = (users: IDeviceUser[]): string[] => {
     : condensed;
 };
 
-const lastSeenTime = (status: string, seenTime: string): string => {
+const lastSeenTime = (status: string, seenTime: string, platform?: string): string => {
```

ESLint

[error] Replace `status:·string,·seenTime:·string,·platform?:·string` with `⏎··status:·string,⏎··seenTime:·string,⏎··platform?:·string⏎`

(prettier/prettier)